src/services/Check.php

```diff
@@ -229,6 +229,7 @@ public static function target(string $target): string
             'title',
             'unreference',
             'userid',
+            'uploadid',
```

Comment:
should be above userid

Reply:
but than I sort the entire list alphabetically

Reply:
isn't it sorted already?

Reply:
Kind of, see https://github.com/elabftw/elabftw/pull/3659/commits/538c3b0146b312ace1167ced36a1af12815b8623